test/common/modules/live_objects_helper.js

```
@@ -171,6 +174,97 @@
+            count,
+          },
+        },
+      };
+
+      return obj;
+    }
+
+    stateOperationMessage(opts) {
+      const { channelName, serial, state } = opts;
+
+      state?.forEach((x, i) => (x.serial = `${serial}:${i}`));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)